src/app/parcels/batch/BatchParcelDataGrid.tsx

```diff
@@ -0,0 +1,267 @@
+        headerName: "Phone Number",
+        width: 150,
+        editable: true,
+    },
+    {
+        field: "address",
+        headerName: "Address",
+        width: 200,
+        editable: true,
+    },
+    {
+        field: "adults",
+        headerName: "Adults",
+        type: "number",
+        width: 110,
+        editable: true,
+    },
+    {
+        field: "children",
+        headerName: "Children",
+        type: "number",
+        width: 110,
+        editable: true,
+    },
+    {
+        field: "listType",
+        headerName: "List Type",
+        width: 150,
+        editable: true,
+    },
+    {
+        field: "dietaryRequirements",
+        headerName: "Dietary Requirements",
+        width: 200,
+        editable: true,
+    },
+    {
+        field: "feminineProducts",
+        headerName: "Feminine Products",
+        width: 150,
+        editable: true,
+    },
+    {
+        field: "babyProducts",
+        headerName: "Baby Products",
+        width: 150,
+        editable: true,
+    },
+    {
+        field: "petFood",
+        headerName: "Pet Food",
+        width: 110,
+        editable: true,
+    },
+    {
+        field: "otherItems",
+        headerName: "Other Items",
+        width: 200,
+        editable: true,
+    },
+    {
+        field: "deliveryInstructions",
+        headerName: "Delivery Instructions",
+        width: 200,
+        editable: true,
+    },
+    {
+        field: "extraInformation",
+        headerName: "Extra Information",
+        width: 200,
+        editable: true,
+    },
+
+    {
+        field: "attentionFlag",
+        headerName: "Attention Flag",
+        width: 150,
+        editable: true,
+    },
+    {
+        field: "signpostingCall",
+        headerName: "Signposting Call",
+        width: 150,
+        editable: true,
+    },
+    {
+        field: "notes",
+        headerName: "Notes",
+        width: 200,
+        editable: true,
+    },
+    {
+        field: "voucherNumber",
+        headerName: "Voucher Number",
+        width: 150,
+        editable: true,
+    },
+    {
+        field: "packingDate",
+        headerName: "Packing Date",
+        width: 150,
+        editable: true,
+    },
+    {
+        field: "packingSlot",
+        headerName: "Packing Slot",
+        width: 150,
+        editable: true,
+    },
+    {
+        field: "shippingMethod",
+        headerName: "Shipping Method",
+        width: 150,
+        editable: true,
+    },
+    {
+        field: "collectionInfo",
+        headerName: "Collection Info",
+        width: 200,
+        editable: true,
+    },
+];
+
+const styledColumns: GridColDef<(typeof mockData)[number]>[] = columns.map((column) => {
+    return { ...column, headerAlign: "center", align: "center" };
+});
+
+const mockData: BatchGridDisplayRow[] = Array.from({ length: 6 }, (_, index) => ({
+    id: index,
+    rowName: index,
+    fullName: "",
+    phoneNumber: "",
+    address: "",
+    adults: null,
```

Comment:
linked to the comment in the props: is there a reason you choose to have the default value as null for numeric values, rather than 0 or -1, but choose to use empty strings for strings?

Reply:
because we want empty cells, this can be an empty string "" or null. for actual data we want adults to be a number, but for an empty row we don't want to display anything (instead of 0 as default for example).

this is specifically for the display of empty cells for a new empty row.